docs/user/DockerInstall.md

```diff
@@ -0,0 +1,44 @@
+# Docker Installation
+
+Install latest docker tool based on OS from [here](https://docs.docker.com/engine/install/)
+
+## Manage Docker as a non-root user
+After Docker engine installation it is mandatory to make docker
+as non-root user to avoid permission issues while running the flow.
+Run below commands to make docker as non-root user.
+
+```
+sudo groupadd docker
+sudo usermod -aG docker $USER
+sudo reboot # REBOOT!
+```
+
+You must restart your operating system for the group permissions to apply.
```

Comment:
As mentioned above, this is not true. The user needs to open a new shell or log off and log in again.